dicom_utils/dicom.py

```diff
@@ -217,9 +230,16 @@ def read_dicom_image(
 
     # in some dicoms, pixel value of 0 indicates white
     # NOTE: apply inversion after window level
-    if is_inverted(dcm.PhotometricInterpretation):
```

Comment:
We can probably drop the old `is_inverted` function

Reply:
Resolved by deprecation